stats/aggregation_data.go

```diff
@@ -74,6 +74,133 @@ func (a *CountData) equal(other AggregationData) bool {
 	return int64(*a) == int64(*a2)
 }
 
+// SumData is the aggregated data for a SumAggregation.
+// A sum aggregation processes data and sums up the recordings.
+//
+// Most users won't directly access sum data.
+type SumData float64
+
+func newSumData(v float64) *SumData {
+	tmp := SumData(v)
+	return &tmp
+}
+
+func (a *SumData) isAggregate() bool { return true }
+
+func (a *SumData) addSample(v interface{}) {
+	// Both float64 and int64 values will be wrapped to float64
+	var f float64
+	switch x := v.(type) {
+	case int64:
+		f = float64(x)
+	case float64:
+		f = x
+	default:
+		return
+	}
+	*a = SumData(float64(*a) + f)
+}
+
+func (a *SumData) multiplyByFraction(fraction float64) AggregationData {
+	return newSumData(float64(*a) * fraction)
+}
+
+func (a *SumData) addToIt(av AggregationData) {
+	other, ok := av.(*SumData)
+	if !ok {
+		return
+	}
+	*a = *a + *other
+}
+
+func (a *SumData) clear() {
+	*a = 0
+}
+
+func (a *SumData) equal(other AggregationData) bool {
+	a2, ok := other.(*SumData)
+	if !ok {
+		return false
+	}
+
+	epsilon := math.Pow10(-9)
```

Comment:
Minor nit, in the future, we can make this a const i.e
```go
const epsilon = 1e-9
```

Reply:
Done.